### ISSUE TITLE
[Bug]: Strange blank border in Fullscreen mode with verticaltab enabled

### ISSUE THREAD

**Issue by anotherwanwanyi** [Floorp-Projects/Floorp#1105]
### Pre-Submission Checklist

- [X] I have checked the existing/closed issues to ensure this issue has not already been reported.
- [X] I am using the latest supported version of the browser.
- [X] I have tried disabling all browser extensions/add-ons to rule out interference from these.
- [ ] I have tried clearing the browser cache and cookies.
- [ ] I have tried to create a new profile and the issue still occurs.
- [X] I have provided detailed step-by-step instructions on how to reproduce the issue.
- [X] I have included relevant screenshots or console outputs.
- [ ] I have checked this problem is not ocurring in the latest version of Firefox ESR.
- [X] This issue is specific to this browser and does not occur in other browsers.

### Bug Summary

With the vertical tab feature enabled, in Fullscreen mode, my browser's webpage's left side has a strange blank border.

### Reproduction Steps

1. Enable vertical tab feature.
2. Open a webpage in Fullscreen mode.

### Expected Behavior

Blank border shouldn't be there.

### Actual Behavior

Fullscreened webpage with a left border.

### Screenshots

Pic-1: 
Actual Behavior: Github issue webpage I'm viewing now.
![图片](https://github.com/Floorp-Projects/Floorp/assets/72936236/1f5dba27-4b3c-4345-991a-e7b43f3b904b)

Pic-2:
Expected Behavior: Without blank border.
![图片](https://github.com/Floorp-Projects/Floorp/assets/72936236/4a882a36-7e07-4faa-9acf-a319a5f500d3)

### Browser Version

11.12.0

### Operating System

Arch Linux

### Additional Context

After some digging, I found the origin of this bug.

It's in the file [native-verticaltab-hover.css](https://github.com/Floorp-Projects/Floorp-core/blob/main/browser/themes/designs/options/native-verticaltab-hover.css).

More specifically, the code is listed below:

```CSS
@media (-moz-platform: linux) {
  ...
  #appcontent {
    margin-left: 27px;
  }
}
```

One possible solution is:

```CSS
:root[inFullscreen]
#appcontent {
    margin-left: 0px !important;
}
```

Adding this code to my `userChrome.css` file resolved the issue for me.

It's worth mentioning that there is a code snippet targeting macOS in that file, which is similar to the code snippet mentioned above:

```CSS
@media (-moz-platform: macos) {
  ...
  #appcontent {
    margin-left: -20px;
  }
}
```

So I suspect there may also be a similar bug on macOS.

**Comment by TheChilledBuffalo** [comment]
@surapunoyousei 
Maybe it is better to revert the change I made that fixes the website resizing problem.... Revert it for now, and I'll write a better fix for it before 11.13.0

The current code has way too many issues....

**Comment by surapunoyousei** [comment]
I still see a lot of compatibility issues.

**Comment by surapunoyousei** [comment]
I revert it

**Comment by JanNowak94** [comment]
I would like to add (as note to any future development) to this bug that it happens not only in fullscreen - if you toggle "**_Show Vertical Tab Bar on the right side of the window_**" it is **always** in this state. Additionally same commit "#TabsToolbar { margin-right: -27px; }" pushes Vertical Tab Bar partially outside of application area - this makes "Close Tab" hard to click on.

**Comment by TheChilledBuffalo** [comment]
@JanNowak94 Really sorry about that. In an attempt to fix #1050 , I introduced a lot of bugs. I found out about those only after the version was released. The margin stays in Fullscreen mode, Sidebar and in PWA windows... I've requested Surapu-san to revert the buggy commits (https://github.com/Floorp-Projects/Floorp-core/commit/620cd8b733f5de4a71cd59ab4adc403a081906b6 and https://github.com/Floorp-Projects/Floorp-core/commit/416abc8e26a2e8e8829c591da9b31143689db630 ) that introduced these bugs and the version containing those should be released very soon. I'll try to find a better solution for the initial issue I was trying to fix before the next major version.
